t/plugin/ext-plugin/http-req-call.t

```diff
@@ -514,8 +514,8 @@ GET /hello
             ext.go({rewrite_bad_path = true})
         }
     }
---- access_log
-GET /plugin_proxy_rewrite_args%3Fa=2
+--- error_log
+undefined path in test server, uri: /plugin_proxy_rewrite_args?a=2
```

Comment:
IMHO, we can use `request_uri` so we know it is a "bad" path.

Reply:
The access log of the test server has been placed in another file. --- access_log It seems that the access_log file path cannot be specified.

Reply:
I mean using it in the error log. Like:
```
ngx.log(ngx.WARN, "undefined path in test server, uri: ", ngx.var.request_uri)
```